common/src/main/resources/assets/wynntils/lang/en_us.json

```diff
@@ -46,8 +46,9 @@
   "feature.wynntils.healthPotionBlocker.threshold.name": "Threshold (%%)",
   "feature.wynntils.healthPotionBlocker.thresholdReached": "You are already at %s%% of your max health!",
   "feature.wynntils.ingredientPouchHotkey.name": "Ingredient Pouch Keybind Feature",
+  "feature.wynntils.itemGuess.levelline": "  §a- §7Lv. %s ",
```

Comment:
'levelLine' perhaps?

Reply:
That would be better

Reply:
Also, by having the space as the last char, we seem to have double space in the output? Is that intentional?

![image](https://user-images.githubusercontent.com/49001742/182679309-8dce8adf-5e93-4ea6-ad50-b0082774d6fe.png)

Reply:
No